Apps/WebClient/src/ClientApp/test/stubs/store/config.ts

```diff
@@ -14,14 +14,16 @@ import {
     ConfigState,
 } from "@/store/modules/config/types";
 
-var configState: ConfigState = {
+import { stubbedPromise, stubbedVoid } from "../../utility/stubUtil";
```

Comment:
instead of .../ lets use @test

Reply:
error TS2307: Cannot find module '@test/utility/stubUtil' or its corresponding type declarations.
not implemented yet or Am i missing something?

Reply:
```@test``` is mapping to  the test folder by ```"@test/*": ["test/*"]``` 
navbar.header.test and monthYearPicker.test uses them

Reply:
I think you added the changes needed to support this feature in #2247, which hasn't been merged to dev yet.